prometheus_client/twisted/_exposition.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+from . import REGISTRY, generate_latest, CONTENT_TYPE_LATEST
+
+from twisted.web.resource import Resource
+
+
+class MetricsResource(Resource):
+    """
+    Twisted ``Resource`` that serves prometheus metrics.
+    """
+    isLeaf = True
+
+    def __init__(self, registry=REGISTRY):
+        self.registry = registry
+
+    def render_GET(self, request):
+        request.setHeader('Content-Type', CONTENT_TYPE_LATEST.encode('ascii'))
+        return generate_latest(self.registry).encode('ascii')
```

Comment:
Sorry. I missed this when doing the changes. I've now tested it against python 3.4 and the tests pass.

Note: the tests (but not the code) need twisted trunk to run (the http client being used was just recently ported).